contracts/src/v0.8/upkeeps/CronUpkeep.sol

```diff
@@ -175,6 +173,10 @@ contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pa
       uint256 nextTick
     )
   {
+    if (!s_activeCronJobIDs.contains(id)) {
```

Comment:
can we add a test for this reversion as well?

Reply:
We already have a test for this on 352 line.